com.arc.embeddedcdt/src/com/arc/embeddedcdt/gui/DebuggerGroupContainer.java

```diff
@@ -253,11 +253,21 @@ public void performApply(ConfigurationWriter configurationWriter) {
         launchExternalNsimInvalidInstructionException);
     configurationWriter.setNsimSimulateExceptions(externalNsimEnableExceptionToolsEnabled);
 
+    if (jtagFrequency != null)
+        configurationWriter.setAshlingJtagFrequency(getAttributeValueFromString(jtagFrequency));
     configurationWriter.setAshlingTDescPath(ashlingTdescPath);
     configurationWriter.setAshlingXmlPath(ashlingXmlPath);
     configurationWriter.setAshlingPath(externalToolsAshlingPath);
   }
 
+  public static String getAttributeValueFromString(String string) {
+    String content = string;
```

Comment:
I'm not sure why this variable is needed here. Could be:
```java
  public static String getAttributeValueFromString(String string) {
    if (string.length() > 0) {
      return string;
    }
    return null;
  }
```

This functions also will fail if `string == null`. Should be something like this:
```java
if (string == null || string.length() > 0) {
  return null;
}
```